core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1920,8 +1930,15 @@ private DefaultServerConfig buildServerConfig(List<ServerPort> serverPorts) {
                 ChannelUtil.applyDefaultChannelOptions(
                         childChannelOptions, idleTimeoutMillis, pingIntervalMillis);
 
-        ServerErrorHandler errorHandler = this.errorHandler;
-        if (errorHandler != ServerErrorHandler.ofDefault()) {
+        ServerErrorHandler errorHandler;
+        if (!exceptionReportingInterval.isZero()) {
```

Comment:
We have a bug here because we didn't normalize the input :wink:

Reply:
Thank you for the careful review 👍